src/components/common/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -226,12 +233,12 @@ export default class NameTranslation extends Mixins(CommonMixin) {
         translation.action = null
         return translation
       })
-    this.emitNameTranslations(this.draftTranslations)
+    this.setNameTranslations(this.draftTranslations)
     this.isEditing = false
   }
 
   protected cancelNameTranslations () {
-    const nameTranslations = this.nameTranslations || []
```

Comment:
This fallback, and also on lines 258 and 340, are unnecessary, since the store value is always defined (initially as []).